### PR TITLE
[Retribution] House Keeping

### PR DESCRIPTION
**Pull request by LordSkeletor**
-Abilities file is possibly typescript now?
-Wake of Ashes module updated to remove talent check
-Updated Fires of Justice to new name
-Moved Consecration from Talents to Core
-Removed GCD rom Avenging Wrath and Crusade
-Added a bunch of missing abilities
-Removed Blessing of Kings and Wisdom

### CODE REVIEW

File: src/parser/paladin/retribution/modules/Abilities.tsx [hunk 1]
```diff
@@ -27,9 +25,7 @@ class Abilities extends CoreAbilities {
         buffSpellId: SPELLS.CRUSADE_TALENT.id,
         cooldown: 120,
         enabled: combatant.hasTalent(SPELLS.CRUSADE_TALENT.id),
-        gcd: {
-          base: 1500,
-        },
+        gcd: undefined,
```
Comment (abelito75) [inline src/parser/paladin/retribution/modules/Abilities.tsx]: If its off gcd you don't need to include this field

File: src/parser/paladin/retribution/modules/Abilities.tsx [hunk 2]
```diff
@@ -43,30 +39,41 @@ class Abilities extends CoreAbilities {
         buffSpellId: SPELLS.AVENGING_WRATH.id,
         cooldown: 120,
         enabled: !combatant.hasTalent(SPELLS.CRUSADE_TALENT.id),
-        gcd: {
-          base: 1500,
-        },
+        gcd: undefined,
```
Comment (abelito75) [inline src/parser/paladin/retribution/modules/Abilities.tsx]: If its off gcd you don't need to include this field

File: src/parser/paladin/retribution/modules/Abilities.tsx [hunk 3]
```diff
@@ -195,6 +199,7 @@ class Abilities extends CoreAbilities {
         isDefensive: true,
         category: Abilities.SPELL_CATEGORIES.UTILITY,
         cooldown: combatant.hasTalent(SPELLS.UNBREAKABLE_SPIRIT_TALENT.id) ? 420 : 600,
+        gcd: undefined,
```
Comment (abelito75) [inline src/parser/paladin/retribution/modules/Abilities.tsx]: If its off gcd you don't need to include this field

File: src/parser/paladin/retribution/modules/Abilities.tsx [hunk 4]
```diff
@@ -243,11 +234,13 @@ class Abilities extends CoreAbilities {
         spell: SPELLS.HAND_OF_RECKONING,
         category: Abilities.SPELL_CATEGORIES.UTILITY,
         cooldown: 8,
+        gcd: undefined,
```
Comment (abelito75) [inline src/parser/paladin/retribution/modules/Abilities.tsx]: If its off gcd you don't need to include this field

File: src/parser/paladin/retribution/modules/Abilities.tsx [hunk 5]
```diff
@@ -243,11 +234,13 @@ class Abilities extends CoreAbilities {
...
       },
       {
         spell: SPELLS.REBUKE,
         category: Abilities.SPELL_CATEGORIES.UTILITY,
         cooldown: 15,
+        gcd: undefined,
```
Comment (abelito75) [inline src/parser/paladin/retribution/modules/Abilities.tsx]: If its off gcd you don't need to include this field

File: src/parser/paladin/retribution/modules/Abilities.tsx [hunk 6]
```diff
@@ -274,10 +267,83 @@ class Abilities extends CoreAbilities {
...
+        channel: (haste: number) => (1.5 / (1 + haste)),
+        gcd: {
+          base: 1500,
+        },
+      },
+      {
+        spell: SPELLS.BLESSING_OF_SACRIFICE,
+        category: Abilities.SPELL_CATEGORIES.UTILITY,
+        cooldown: 120,
+        gcd: undefined,
```
Comment (abelito75) [inline src/parser/paladin/retribution/modules/Abilities.tsx]: If its off gcd you don't need to include this field

File: src/parser/paladin/retribution/modules/Abilities.tsx [hunk 7]
```diff
@@ -274,10 +267,83 @@ class Abilities extends CoreAbilities {
...
+        cooldown: 30,
+        gcd: {
+          base: 1500,
+        },
+      },
+      {
+        spell: SPELLS.SHIELD_OF_THE_RIGHTEOUS,
+        category: Abilities.SPELL_CATEGORIES.UTILITY,
+        cooldown: 1,
+        gcd: undefined,
```
Comment (abelito75) [inline src/parser/paladin/retribution/modules/Abilities.tsx]: If its off gcd you don't need to include this field

File: src/parser/paladin/retribution/modules/core/WakeofAshes.js [hunk 8]
```diff
@@ -23,10 +23,6 @@ class WakeofAshes extends Analyzer {
 
     constructor(...args) {
         super(...args);
-        this.active = this.selectedCombatant.hasTalent(SPELLS.WAKE_OF_ASHES.id);
-        if (!this.active) {
-            return;
-        }
         this.addEventListener(Events.damage.by(SELECTED_PLAYER).spell(SPELLS.WAKE_OF_ASHES), this.onWakeofAshesDamage);
```
Comment (abelito75) [inline src/parser/paladin/retribution/modules/core/WakeofAshes.js:26]: I can't comment directly on it 
but you should probably change the STATISTIC_ORDER to CORE from UNIMPORTANT()

**Review by abelito75 — CHANGES_REQUESTED** [review]
Just small things but overall looking good

**Comment by LordSkeletor** [comment]
-Removed undefined GCD
-Added Wake and Consecration module to Core Statistics

**Review by abelito75 — APPROVED** [review]

**Pull request merged**